packages/ffe-cards-react/src/IconCard/IconCard.js

```diff
@@ -28,6 +28,8 @@ const IconCard = props => {
                     'ffe-icon-card__icon',
                     icon.props.className,
                 ),
+                color: 'ffe-farge-fjell',
```

Comment:
Jeg ville definert fargen i CSS, mye lettere å arbeide med og få darkmode til å fungere.  Jo mer jeg jobber meg gjennom komponentene jo mindre ser jeg behovet for den color-propertyen, så den kommer nok til å bli fjernet før ordentlig release.  Ingen grunn til å legge til mer inline styling enn nødvendig.

Reply:
Jag gjorde ett nytt førsøk